src/gui/inputdevices/StylusInputHandler.cpp

```diff
@@ -110,7 +127,9 @@ void StylusInputHandler::setPressedState(InputEvent const& event) {
     {
         switch (event.button) {
             case 1:
-                this->deviceClassPressed = true;
+                if (!this->eventsToIgnore >= 0) {
+                    this->deviceClassPressed = true;
+                }
```

Comment:
Looks like a typo here? As it is currently written, `!this->eventsToIgnore` is the same as `this->eventsToIgnore != 0` but then `>= 0` always evaluates to true.